auth/src/main/res/values/styles.xml

```diff
@@ -2,17 +2,10 @@
 <resources>
 
     <style name="FirebaseUI" parent="Theme.AppCompat.Light.DarkActionBar">
-        <item name="colorPrimary">@color/authui_colorPrimary</item>
```

Comment:
So are all of the deleted values OK because they default to relying on `colorPrimary`, `colorPrimaryDark`, and `colorAccent`?